scripts/syseeprom-to-json

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/awk -f
```

Comment:
Can you please explain, how is this used?

Reply:
This is a utility to convert the text output of the syseeprom dump utility into JSON.